lib/internal/Magento/Framework/Session/SessionManager.php

```diff
@@ -5,6 +5,9 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
+// @codingStandardsIgnoreFile
```

Comment:
It's not recommended to suppress warnings in all new and updated code. For this specific case I would suggest narrowing down the ignore section to the specific lines, indicated by the test.